protocol/chat.go

```diff
@@ -60,15 +60,17 @@ const (
 	MuteFor1WeekDuration  = 7 * 24 * time.Hour
 )
 
+// NOTE: Add items to the end of the list, because desktop and mobile
+// use this enum by number rater than by string.
```

Comment:
Tricky case, another fix would be to adapt clients.

PS. Protobufs as API types would prevent such issues.